online-inference/hf-llm/Dockerfile

```
@@ -0,0 +1,13 @@
+FROM ghcr.io/coreweave/ml-containers/torch:afecfe9-base-cuda11.8.0-torch2.0.0-vision0.15.1
```

The `ml-containers` Dockerfile currently uses torch-base with CUDA 12.0.1 (we could update these to a newer build too)

```suggestion
FROM ghcr.io/coreweave/ml-containers/torch:afecfe9-base-cuda12.0.1-torch2.0.0-vision0.15.1
```